src/js/tabs/pid_tuning.js

```diff
@@ -408,20 +404,16 @@ TABS.pid_tuning.initialize = function (callback) {
                     }
                     $('.dMinDisabledNote').hide();
                     $('.dminGroup .suboption').show();
-                    $('#pid_main tr :nth-child(5)').show();
-                    $('#pid_main .pid_titlebar2 th').attr('colspan', 6);
-                    $('.helpicon[i18n_title="pidTuningPidTuningTipFeedforward"]').hide();
-                    $('.helpicon[i18n_title="pidTuningPidTuningTipDMin"]').show();
+                    //$('#pid_main tr :nth-child(5)').show();
```

Comment:
Can you please remove commented out code, or else add a comment stating why this was commented out?

Reply:
It was to let current testers test out D_min OFF/ON, so the values are shown instead of hiding the whole D_min column. This will be uncommented for final version.

Reply:
This seems to be getting closer to being mergeable, so these should be uncommented now I guess.